tasks/RedHat/generic/assert-tsx.yml

```diff
@@ -4,6 +4,24 @@
     msg: "imported RedHat/generic/assert-tsx.yml"
 
 - block:
+# There are CPUs which are not capable of enabling the rtm flag, so we just report the status:
+    - name: Get all CPU flags
+      shell: lscpu | grep "^Flags:"
+      register: __sap_hana_preconfigure_register_lscpu_flags
+      changed_when: no
+
+    - name: Report that the rtm CPU flag exists
+      debug:
+        msg: "INFO: The CPU flags contain ' rtm'."
+      when: "' rtm' in __sap_hana_preconfigure_register_lscpu_flags.stdout"
```

Comment:
are you guaranteed that `rtm` will never be the first cpu flag listed?  that is, are you guaranteed that the cpu flag `rtm` will always be preceded by a space in the output?

Reply:
There has never been a case where `rtm` was the first cpu flag. However, I cannot exclude it. So I will find a different solution. What I wanted to avoid is that we get a hit if there will ever be a future flag which has a substring `rtm`.
I created https://github.com/linux-system-roles/sap-hana-preconfigure/issues/198 for tracking this issue. For the time being, I'd like to merge this change. It has also passed QA.